myuw/templates/handlebars/card/directory_info.html

```diff
@@ -59,9 +41,6 @@ <h4 class="myuw-card-row-heading">Email 2</h4>
                 </div>
                 {{/if}}
 
-            </div>
-
-            <div class="myuw-card-section" property="workLocation" typeof="Place">
 
                 <span class="sr-only" property="name">UW Office</span>
```

Comment:
Let's put the `<span class="sr-only">` here inside the `<div class="col-sm-3">` as the first element. Where it is placed now is causing some inconsistent spacing due to how the CSS is written.